src/main/pg/sdcard.c

```diff
@@ -47,6 +47,11 @@ void pgResetFn_sdcardConfig(sdcardConfig_t *config)
     config->device = SPI_DEV_TO_CFG(SPIINVALID);
     config->mode = SDCARD_MODE_NONE;
 
+#if defined(STM32H7) && defined(USE_SDCARD_SDIO) // H7 only for now, likely should be applied to F4/F7 too
+    config->mode = SDCARD_MODE_SDIO;
```

Comment:
Why? This will break all kinds of things once a Unified Target for H7 is introduced.

Reply:
See above.

Reply:
If using the SDMMC engine on an H7 (SDIO) then it has its own DMA engine, so this code says to use it. The DMA option is only here to give configurability to ensure D-Shot can get enough descriptors and that's not an issue in this case. So, if SDIO is configured on an H7 this code just records that fact and safely enables DMA.

Reply:
Still, this has to be done through each individual's target configuration, and not in here.

Reply:
Doh! It was already in the SPRACINGH7ZERO and SPRACINGH7EXTREME config.c files. The `sdcardConfigMutable()->useDma = true` was there too, which it needn't have been because I realise the sdcard_dma setting is only used if USE_SDCARD_SPI is defined, which of course it isn't here.